ci/kind/test-e2e-kind.sh

```diff
@@ -235,11 +253,11 @@ done
 # The Antrea images should not be pulled, as we want to use the local build.
 if $coverage; then
     manifest_args="$manifest_args --coverage"
-    COMMON_IMAGES_LIST+=("antrea/antrea-agent-ubuntu-coverage:latest" \
-                         "antrea/antrea-controller-ubuntu-coverage:latest")
+    COMMON_IMAGES_LIST+=("${antrea_controller_image}-coverage:${antrea_image_tag}" \
+                         "${antrea_agent_image}-coverage:${antrea_image_tag}")
```

Comment:
it's a bit weird to append the `-coverage` suffix automatically when a user provides an image names.
I don't have a great solution, but I think we should fail the script if the `--coverage` flag is used with `--antrea-controller-image` or `--antrea-agent-image`.

Reply:
Added simple checks to make sure image names cannot be specified with coverage enabled.